src/rtapi/uspace_rtapi_app.cc

```diff
@@ -919,6 +922,7 @@ int Posix::task_delete(int id)
 static int find_rt_cpu_number() {
     if(getenv("RTAPI_CPU_NUMBER")) return atoi(getenv("RTAPI_CPU_NUMBER"));
 
+#ifdef __linux__
     cpu_set_t cpuset_orig;
     int r = sched_getaffinity(getpid(), sizeof(cpuset_orig), &cpuset_orig);
```

Comment:
maybe cpuset_get/setaffinity, when it's useful in the future. but yea, it looks like two implementations will be needed.

https://www.freebsd.org/cgi/man.cgi?query=cpuset_getaffinity&sektion=2&apropos=0&manpath=FreeBSD+7.1-RELEASE